content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -162,7 +162,8 @@ The Patient Resource supports proprietary codes for:
 The RelatedPerson Resource supports proprietary codes for:
 
 * RelatedPerson.communication.language codes are maintained in [Code Set 36 Languages](#code-set-36-languages)
-* RelatedPerson.relationship codes are maintained in [Code Set 40 Person Relationship Types](#code-set-40-person-relationships) and [Code Set 351 Person Relationship Types](#code-set-351-person-relationship-types)
+* RelatedPerson.relationship codes are maintained in [Code Set 351 Person Relationship Types](#code-set-351-person-relationship-types)
+* RelatedPerson.relationship.relation codes are maintained in [Code Set 40 Person Relationship Types](#code-set-40-person-relationships)
```

Comment:
Nit I think we should make this something along the lines of `The relation extensions on RelatedPerson.relationship are maintained...`. We do that format in other places: https://github.com/cerner/fhir.cerner.com/blame/e58e6b32556b979db32453b777872b9877aadebe/content/millennium/r4/proprietary-codes-and-systems.md#L123

Reply:
+1 please capitalize Relation to make it "The relation extensions on RelatedPerson.relationship are maintained..."

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/644/commits/767fd43d02089bf605dfdde72918fc393bec5655#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R166

Reply:
+1  to this change.